workflow/controller/operator.go

```diff
@@ -956,8 +956,8 @@ func (woc *wfOperationCtx) assessNodeStatus(pod *apiv1.Pod, node *wfv1.NodeStatu
 	case apiv1.PodRunning:
 		if pod.DeletionTimestamp != nil {
 			// pod is being terminated
-			newPhase = wfv1.NodeFailed
-			message = "pod termination"
+			newPhase = wfv1.NodeError
+			message = "pod deleted during operation"
```

Comment:
I believe this needs to be reverted as well since you will be reverting the logic of deletion of running pods..

Reply:
This change is unrelated. This makes sure that Nodes that are in which pods are deleted out-of-band get marked as `Error` instead of `Failed`

Reply:
It's simply a node label change